assets/environment/table_description/package.xml

```diff
@@ -4,8 +4,8 @@
   <name>table_description</name>
   <version>2.0.0</version>
   <description>table_description description</description>
-  <maintainer email="example@gmail.com">name</maintainer>
-  <license>TODO: License declaration</license>
+  <maintainer email="glenn_tan@artc.a-star.edu.sg">Tan Jing Peng GLenn</maintainer>
+  <license>Apache License 2.0</license>
```

Comment:
@cardboardcode  It's already changed here. The link that you reference is to an old commit.

Reply:
Yup. Apologies. Was looking at the outdated file.